decentraland/unity-explorer
# fix: Support for multiple NFT chains

## What does this PR change?
Fix #1475 

- This fix adds support for every chain (Ethereum, Matic, etc.) that [OpenSea supports](https://docs.opensea.io/reference/get_nft) to be able to show it in our NFT Dialogs.

![image](https://github.com/user-attachments/assets/1d23fcf1-05e2-492b-ac54-238efb0cd432)

![image](https://github.com/user-attachments/assets/b4b91326-13d5-4c10-aa07-470a12f110f1)

## How to test the changes?
1. Launch the explorer
2. `/goto goerli`
3. Navigate to coordinates (85, -8) in the Goerli NFT-Museum scene.
4. Click on any NFT in the walls (in any floor) and check that they're opening correctly.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
